### ISSUE TITLE
Make error

### ISSUE THREAD

**Issue by halilozercan** [viq854/balaur#4]
When I try to compile the project on fresh install of Ubuntu 16.04, I get the following error.

```make: *** No rule to make target 'obj/main.o', needed by 'totoro'.  Stop.```

totoro is defined in Makefile as PROG.

**Comment by jdecouchant** [comment]
Yes, I had the same error. The makefile is broken. I modified it so that it would start compiling (but if fails later due to other reasons).
